quickstart.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Quick Start
+---
+
+# Quick Start
+
+For a start, let's build an "echo" bot in python. For every message you send to it, it will reply with that same text.
+You can look at this [example repository](https://github.com/deltachat-bot/echo) for solutions in different programming languages!
+{: .box }
+
+To install the `deltachat` python package, see the [deltachat python
+documentation](https://py.delta.chat/install.html). You can import it like
+this:
+
+```python
+import deltachat
+```
+
+## Setting up an Account Object
+
+A Delta Chat usually starts with an account object, the account your bot will
+use to communicate with others. It is initialized with a path to the database
+where it will store its data. Make sure that path exists:
+
+```python
+import deltachat
+
+ac = deltachat.account.Account("/tmp/bot_db/")
+```
+
+Now, to login to the email account, the bot needs its own email address + a
+password. With most servers, this will be enough to get started (some servers
+might require [additional configuration](https://providers.delta.chat)). Let's
+get username + password from environment variables, configure, and run the
+account:
+
+```python
+import deltachat
+import os
+
+os.getenv("DELTACHAT_DB_PATH", default="/tmp/bot_db/")
+ac = deltachat.account.Account(db_path)
+
+addr = os.getenv("DELTACHAT_BOT_ADDR")
+password = os.getenv("DELTACHAT_BOT_PASSWORD")
+
+ac.run_account(addr=addr, password=password)
+ac.wait_shutdown()
+```
+
+[`account.run_account()`](https://py.delta.chat/api.html#deltachat.account.Account.run_account)
+will run the account, and configure it if necessary (e.g. look up the mail
+servers of your provider).
+
+[`account.wait_shutdown()`](https://py.delta.chat/api.html#deltachat.account.Account.wait_shutdown)
+waits until the account quits running for some reason. This is necessary here
+so the script doesn't quit directly, but stays open as long as the bot is
+running. We want the bot to reply to our messages forever, after all.
```

Comment:
maybe move the text to comments and just have a list of links after it? If we even need the links to the code reference?

Reply:
They are now not directly after each other, but are explaining a code snippet directly - I think it makes sense now as it is.